news/2023/2023-03-25-new-featured-artist-ashrount.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "New Featured Artist: Ashrount"
+date: 2023-03-25 11:00:00 +0000
+---
+
+**Ashrount** pounds through the doors to our Featured Artist library.
+
+![](https://assets.ppy.sh/artists/352/header.jpg)
+
+You're more familiar with **Ashrount** than you may think. While their newest productions are under the alias **Ashrount**, they also produced [Emptiness](https://osu.ppy.sh/beatmapsets/1210804) under the name *Xect.* for BOFU2017, and a heap of older songs under the moniker *Maverick*, like the multi-mode marathon set [Maverick - Shironaga Star Jet -aquamarine mix-](https://osu.ppy.sh/beatmapsets/1012634).
+
+Regardless of how you know **Ashrount**, it's clear that their songs are in a league of their own. 
+
+Pick your favourite from the **10** listed on [**Ashrount**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/352) whenever you're ready to begin your mapping journey!
+
+Check everything out for yourself:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/352/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Maverick - opaque
+
+Try out [the map from the video above](https://osu.ppy.sh/beatmapsets/1955283) hosted by [Niva](https://osu.ppy.sh/users/197805)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/352/Songs/Maverick%20-%20opaque.mp3">
+</audio>
+
+### Ashrount - Cataraxir
+
+Anyone who followed Corsace 2020 would recognize [this map](https://osu.ppy.sh/beatmapsets/1276331) created by [Realazy](https://osu.ppy.sh/users/918297)!
```

Comment:
brilliant, i can't check this due to the website not having past mappools and there being no wiki page either

forum page doesn't even link spreadsheets

beatmap description says "corsace grandfinals - NM3", i'll just trust that this refers to the correct tournament

```suggestion
Anyone who followed [Corsace Open 2020](https://osu.ppy.sh/community/forums/topics/1117976) would recognise [this map](https://osu.ppy.sh/beatmapsets/1276331) created by [Realazy](https://osu.ppy.sh/users/918297)!
```